include/NovelRT/Graphics/Metal/Utilities/Graphics.Metal.Utilities.h

```diff
@@ -0,0 +1,17 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_GRAPHICS_METAL_UTILITIES_H
+#define NOVELRT_GRAPHICS_METAL_UTILITIES_H
+
+#ifdef TARGET_OS_MAC
```

Comment:
Let's move this to NoevlRT.h - same reasoning as the D3D12 preprocessor being moved